routes/api/search/volunteer-search.js

```diff
@@ -7,5 +7,24 @@ var router = express.Router();
 
 console.log('volunteer-search');
 
+
+app.get("/api/volunteers", function(req, res) {
+    // 1. Add a join to include all of users
+    db.User.findAll({}).then(function(dbVolunteer) {
+      res.json(dbVolunteer);
+    });
+  });
+
+  app.get("/api/volunteers/:id", function(req, res) {
+    // 2; Add a join to include all of users
+    db.Author.findOne({
+      where: {
+        id: req.params.id
+      }
+    }).then(function(dbVolunteer) {
+      res.json(dbVolunteer);
+    });
+  });
+
```

Comment:
We'll be deleting this whole file and using `read-browse.js` instead since requests and volunteers are handled in the same view now